tests/cases/standalone/common/show/show_index.result

```diff
@@ -38,23 +41,27 @@ SHOW INDEX FROM test;
+| system_metrics | 1          | TIME INDEX     | 1            | ts          | A         |             |          |        | NO   | greptime-inverted-index-v1 |         |               | YES     |            |
++----------------+------------+----------------+--------------+-------------+-----------+-------------+----------+--------+------+----------------------------+---------+---------------+---------+------------+
 
 SHOW INDEX FROM system_metrics in public;
 
-+----------------+------------+------------+--------------+-------------+-----------+-------------+----------+--------+------+----------------------------+---------+---------------+---------+------------+
-| Table          | Non_unique | Key_name   | Seq_in_index | Column_name | Collation | Cardinality | Sub_part | Packed | Null | Index_type                 | Comment | Index_comment | Visible | Expression |
-+----------------+------------+------------+--------------+-------------+-----------+-------------+----------+--------+------+----------------------------+---------+---------------+---------+------------+
-| system_metrics | 1          | PRIMARY    | 1            | host        | A         |             |          |        | YES  | greptime-inverted-index-v1 |         |               | YES     |            |
-| system_metrics | 1          | PRIMARY    | 2            | idc         | A         |             |          |        | YES  | greptime-inverted-index-v1 |         |               | YES     |            |
-| system_metrics | 1          | TIME INDEX | 1            | ts          | A         |             |          |        | NO   | greptime-inverted-index-v1 |         |               | YES     |            |
-+----------------+------------+------------+--------------+-------------+-----------+-------------+----------+--------+------+----------------------------+---------+---------------+---------+------------+
++----------------+------------+----------------+--------------+-------------+-----------+-------------+----------+--------+------+----------------------------+---------+---------------+---------+------------+
+| Table          | Non_unique | Key_name       | Seq_in_index | Column_name | Collation | Cardinality | Sub_part | Packed | Null | Index_type                 | Comment | Index_comment | Visible | Expression |
++----------------+------------+----------------+--------------+-------------+-----------+-------------+----------+--------+------+----------------------------+---------+---------------+---------+------------+
+| system_metrics | 1          | INVERTED INDEX | 6            | desc1       | A         |             |          |        | YES  | greptime-inverted-index-v1 |         |               | YES     |            |
+| system_metrics | 1          | FULLTEXT INDEX | 7            | desc2       | A         |             |          |        | YES  | greptime-inverted-index-v1 |         |               | YES     |            |
```

Comment:
The index type of the fulltext index should not be `greptime-inverted-index-v1`

Reply:
thanks i modify the show index datafusion query to display full text index type as `greptime-fulltext-index-v1`